### ISSUE TITLE
Why title CSS isn't available in documentation ?

### ISSUE THREAD

**Issue by seven7hell** [archriss/react-native-snap-carousel#688]
I used the same code as provided in the documentation but the text is out the image container. When I debug the code, then I get to know that there is not CSS for the title. 

`<Text style={styles.title} numberOfLines={2}>
                    { item.name }
                </Text>`
Can anyone please provide me the css so that I can get the look as mentioned in the example GIF ?

**Comment by dohooo** [comment]
Sorry, please allow me to advertise for my open source library! ~
I think this library [react-native-reanimated-carousel](https://github.com/zwh1666258377/react-native-reanimated-carousel) will solve your problem. It is a high performance and very simple component, complete with React-Native reanimated 2

**Comment by cloudchallengers** [comment]
Yes, I am also facing issue while displaying text over the image with ParallaxImage. 

Can the CSS or an example to display some text over the image be added to this library ?
